src/app/client/src/app/modules/groups/interfaces/group.ts

```diff
@@ -81,3 +81,9 @@ export interface IGroupMember {
   id?: string;
 }
 
+export interface IGroupData {
+  name: string;
```

Comment:
Why the creator & groupId names are difference here than what we get in activated route params?

Reply:
not required. changed the code.